engine/src/main/java/org/hibernate/validator/internal/cfg/context/TypeConstraintMappingContextImpl.java

```diff
@@ -74,9 +73,12 @@
 	private List<Class<?>> defaultGroupSequence;
 	private Class<? extends DefaultGroupSequenceProvider<? super C>> defaultGroupSequenceProviderClass;
 
-	TypeConstraintMappingContextImpl(DefaultConstraintMapping mapping, Class<C> beanClass) {
+	private final GetterPropertyMatcher getterPropertyMatcher;
```

Comment:
I know I already made you change the name but I'm wondering if `GetterPropertySelectionStrategy` would be better?